doc/TOR.md

```diff
@@ -86,7 +86,9 @@ always-use-proxy=true
     for a `SocksPort` entry to confirm the port number.
 2.  `bind-addr` informs C-Lightning to bind itself to port 9735.
     This is needed for the subsequent `statictor` to work.
-    9735 is the normal Lightning Network port.
+    9735 is the normal Lightning Network port, so this setting may already be present.
```

Comment:
Why this "may" be present? Where?

For instance, "can you want to bind-add on more onion address but this is not possible .... other things"